### ISSUE TITLE
(aws-wafv2): customResponseBodies is not present on CfnWebACLProps 

### ISSUE THREAD

**Issue by mschenke** [aws/aws-cdk#14199]
### The Question

CloudFormation documentation shows support for CustomResponseBodies here https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-wafv2-webacl.html#cfn-wafv2-webacl-customresponsebodies

This is missing from the CfnWebACL

**Comment by mschenke** [comment]
CustomResponseBody seems to be present here: https://d1uauaxba7bl26.cloudfront.net/latest/gzip/CloudFormationResourceSpecification.json

**Comment by njlynch** [comment]
Hi @mschenke .

`CustomResponseBodies` was added to the CloudFormation specification in version 32.0.0. The latest merged into the CDK is currently 31.1.0. There is a pending pull request to merge in version 33.0.0, which will bring in the new properties: 
https://github.com/aws/aws-cdk/pull/14168

Once the above is merged in and released, CfnWebACL will be updated. I'm going to close this issue in favor of that one, please mention me or post in that PR if you have any related follow-ups!
